src/sentry/static/sentry/app/views/settings/components/forms/formField/index.tsx

```diff
@@ -14,53 +15,23 @@ import Field from 'app/views/settings/components/forms/field';
 import FieldControl from 'app/views/settings/components/forms/field/fieldControl';
 import FieldErrorReason from 'app/views/settings/components/forms/field/fieldErrorReason';
 import FormFieldControlState from 'app/views/settings/components/forms/formField/controlState';
-import FormModel from 'app/views/settings/components/forms/model';
+import FormModel, {MockModel} from 'app/views/settings/components/forms/model';
 import ReturnButton from 'app/views/settings/components/forms/returnButton';
 
+import {FieldValue} from '../type';
+
 /**
  * Some fields don't need to implement their own onChange handlers, in
  * which case we will receive an Event, but if they do we should handle
  * the case where they return a value as the first argument.
  */
-const getValueFromEvent = (valueOrEvent?, e?: MouseEvent) => {
+const getValueFromEvent = (valueOrEvent?: FieldValue | MouseEvent, e?: MouseEvent) => {
   const event = e || valueOrEvent;
   const value = defined(e) ? valueOrEvent : event && event.target && event.target.value;
 
-  return {
-    value,
-    event,
-  };
+  return {value, event};
 };
 
-// MockedModel that returns values from props
-// Disables a lot of functionality but allows you to use fields
-// without wrapping them in a Form
-class MockModel {
-  //TODO(TS)
-  props: any;
-  initialData: object;
-  constructor(props) {
-    this.props = props;
-
-    this.initialData = {
-      [props.name]: props.value,
-    };
-  }
-  setValue() {}
-  setFieldDescriptor() {}
-  removeField() {}
-  handleBlurField() {}
-  getValue() {
-    return this.props.value;
-  }
-  getError() {
-    return this.props.error;
-  }
-  getFieldState() {
-    return false;
-  }
-}
-
```

Comment:
Moved this into formModel.tsx, just to co-locate it with the thing it's mocking.